lib/drawNametag.ts

```diff
@@ -1,14 +1,26 @@
 // Under Review
 
-export default function drawNametag(nameTagStatus:boolean, currentNameTag:string[], selectedWaveHand: number | null, waveHands: string[]): ImageData {
-  console.log('function received:', nameTagStatus, selectedWaveHand)
+export interface NameTagBadge {
+  visible: boolean;
+  fullName: string;
+  preferredName: string;
+  pronouns: string;
+  disclosure: string;
+}
+export interface HandWaveBadge {
+  visible: boolean;
+  waveText: string | null;
+}
```

Comment:
You can actually include the fact that disabled badges don't need text in the types themselves, like:
```suggestion
interface DisabledBadge {
  visible: false
}
interface EnabledHandWaveBadge {
  visible: true;
  waveText: string;
}
export type HandWaveBadge = DisabledBadge | EnabledHandWaveBadge;
```